stats/handlers.go

```diff
@@ -62,3 +65,8 @@ type Handler interface {
 	// HandleConn processes the Conn stats.
 	HandleConn(context.Context, ConnStats)
 }
+
+type Handler interface {
```

Comment:
Travis is failing because there's no comment here.  You can keep the old comment.